huaweicloud/resource_huaweicloud_cce_addon_v3.go

```diff
@@ -0,0 +1,205 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/cce/v3/addons"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceCCEAddonV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCCEAddonV3Create,
+		Read:   resourceCCEAddonV3Read,
+		Delete: resourceCCEAddonV3Delete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
```

Comment:
as we need two parameters: addon id and cluster id, so we can not import the resource directly.